Source/WebCore/platform/ios/VideoFullscreenInterfaceAVKit.mm

```diff
@@ -56,18 +57,23 @@
 #import <pal/cocoa/AVFoundationSoftLink.h>
 #import <pal/ios/UIKitSoftLink.h>
 
-#if !PLATFORM(WATCHOS)
+#if !PLATFORM(WATCHOS) && !HAVE(PIP_CONTROLLER)
 static const NSTimeInterval playbackControlsVisibleDurationAfterResettingVideoSource = 1.0;
 #endif
 
 SOFTLINK_AVKIT_FRAMEWORK()
 #if HAVE(AVOBSERVATIONCONTROLLER)
 SOFT_LINK_CLASS_OPTIONAL(AVKit, AVObservationController)
 #endif
-SOFT_LINK_CLASS_OPTIONAL(AVKit, AVPictureInPictureController)
 SOFT_LINK_CLASS_OPTIONAL(AVKit, AVPlayerViewController)
 SOFT_LINK_CLASS_OPTIONAL(AVKit, __AVPlayerLayerView)
 
+#if HAVE(PIP_CONTROLLER)
+SOFT_LINK_CLASS_OPTIONAL(AVKit, AVPictureInPictureController)
+SOFT_LINK_CLASS_OPTIONAL(AVKit, AVPictureInPictureControllerContentSource)
+SOFT_LINK_CLASS_OPTIONAL(AVKit, AVPictureInPictureContentViewController)
```

Comment:
I'd really like to know why GitHub syntax highlights only the first macro in a set.